### PR TITLE
Set code formatter job to only trigger on phrase

### PR DESCRIPTION
**Pull request by mmitche**
Also some general cleanup and refactoring to use new APIs

### CODE REVIEW

**Comment by mmitche** [comment]
test ci please

**Comment by stephentoub** [comment]
cc: @sokket

File: netci.groovy [hunk 1]
```diff
@@ -8,51 +8,53 @@ def project = 'dotnet/corefx'
...
+[true, false].each { isPR -> 
+    def newJob = job(Utilities.getFullJobName(project, 'native_code_format_check', isPR)) {
+        steps {
+            shell('python src/Native/format-code.py checkonly')
+        }
+    }
+    
+    // Set up standard options.
+    Utilities.standardJobSetup(newJob, project, isPR)
+    // Set the machine affinity to windows machines
```
Comment (sokket) [inline netci.groovy]: NIT - comment is incorrect, the machine affinity is Ubuntu

**Comment by sokket** [comment]
One minor NIT, otherwise LGTM

**Comment by mmitche** [comment]
test ci please

**Pull request merged**